lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -130,9 +130,22 @@ def test_score_set_signal_handler(self, handler, kwargs, earned, possible):
             'modified': FROZEN_NOW_TIMESTAMP,
             'score_db_table': 'submissions',
         }
+        if handler == submissions_score_reset_handler:
+            expected_set_kwargs['score_deleted'] = True
         self.signal_mock.assert_called_once_with(**expected_set_kwargs)
         self.get_user_mock.assert_called_once_with(kwargs['anonymous_user_id'])
 
+    def test_tnl_6599(self):
```

Comment:
A most excellent test name. Now we're guaranteed to go back and fix this properly.